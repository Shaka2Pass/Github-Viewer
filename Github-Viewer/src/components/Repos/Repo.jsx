import React from 'react';
import PropTypes from 'prop-types';

const Repo = ({ name, url }) => (
  <div>
    <div>
      Name: {name}
    </div>
    <div>
      Url: {url}
    </div>
  </div>
);

Repo.propTypes = {
  name: PropTypes.string.isRequired,
  url: PropTypes.string.isRequired, 
};

export default Repo;
