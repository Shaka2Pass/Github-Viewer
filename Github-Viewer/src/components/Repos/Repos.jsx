import React from 'react';
import PropTypes from 'prop-types';

const Repos = ({ repos }) => (
  <header>
    <div>
      <p>{ repos }</p>
    </div>
  </header>
);

Repos.propTypes = {
  repos: PropTypes.string.isRequired,
};

export default Repos;


