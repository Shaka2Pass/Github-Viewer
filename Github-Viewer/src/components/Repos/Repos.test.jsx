import React from 'react';
import { shallow } from 'enzyme';
import Repos from './Repos';

describe('Repos', () => {
  it('matches a repo', () => {
    const wrapper = shallow(<Repos repos="somerepo"/>);
    expect(wrapper).toMatchSnapshot();
  });
});
