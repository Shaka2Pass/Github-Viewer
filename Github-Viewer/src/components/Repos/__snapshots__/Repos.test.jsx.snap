// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Repos matches a repo 1`] = `
<header>
  <div>
    <p>
      somerepo
    </p>
  </div>
</header>
`;
